assets/scripts/datadog-docs.js

```diff
@@ -416,3 +417,26 @@ window.addEventListener(
     false
 );
 
+
+
+function scrollActiveNavItemToTop(){
+    // Scroll the open top level left nav item into view below Docs search input
+    if (document.querySelector('.sidenav:not(.sidenav-api)')) {
+        const headerHeight = document.querySelector('body .main-nav').style.height;
+        const padding = 200;
+
+        // set max height of side nav. vanilla js here doesnt enable you to set the max height
```

Comment:
hey, @StefonSimmons!    [there is a vanilla JS `maxHeight` property](https://www.w3schools.com/jsref/prop_style_maxheight.asp), did you try using that?   it would be great to avoid adding more jQuery.

Reply:
yep. I did it wouldn't allow me to set maxHeight. it was very strange

Reply:
I was experiencing strange behavior with setting the scrollTop value on page load. I then noticed that  the maxheight was not being set correctly with my vanilla  El.style.maxHeight = value
when I switched to setting the maxHeight with jquery I was able to set scrollTop on page load

Reply:
setting the max-height to a valid value ( pixel ) worked.